library/clinical_rules.php

```diff
@@ -697,11 +697,11 @@ function test_rules_clinic($provider='',$type='',$dateTarget='',$mode='',$patien
   }
 
   // Collect applicable patient pids in only medicare
-if (strpos($type, 'pqrs_individual') !== false ) {
-	$onlyMedicarePatients=true;	
-} else {
+//if (strpos($type, 'pqrs_individual') !== false ) {
```

Comment:
Will these need to be added back in?  If not let's just drop the comment out lines...  or make a note about why they are being kept

Reply:
Actually, we will have to revamp an number of these things immediately for ACO accessibility.  Working on it.